Tasks/IISWebAppDeployment/task.json

```diff
@@ -284,13 +284,13 @@
             "visibleRule": "AppPoolIdentity = SpecificUser"
         },
         {
-            "name": "AppCmdArgs",
-            "type": "string",
-            "label": "Additional Appcmd.exe Arguments",
+            "name": "AppCmdCommands",
```

Comment:
Till now I haven't added IIS Tasks to compact tests.. Did anyone else add it ?? Anyways I will follow up on this and get a closure